aocd/utils.py

```diff
@@ -11,12 +11,20 @@
 from importlib.metadata import entry_points
 from importlib.metadata import version
 from itertools import cycle
+from pathlib import Path
 from tempfile import NamedTemporaryFile
+from typing import Mapping
+from typing import Optional
+from typing import Union
 from zoneinfo import ZoneInfo
 
 import bs4
 import urllib3
 
+from ._compat import get_plugins as get_plugins # import using same name to tell the type checker we intend to export this (so other modules can import it)
```

Comment:
The comment is overly verbose. Why not just use
```suggestion
from ._compat import get_plugins as get_plugins # explicitly re-export to public API
```